BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -1009,7 +1009,11 @@ IEnumerator DogfightCompetitionModeRoutine(float distance, bool startDespiteFail
                         if (pilot == null) continue;
 
                         if (!pilot.weaponManager.guardMode)
+                        {
```

Comment:
This would be better in the `SpawnUtilsInstance.ApplyRWP` function.

Reply:
This also means we need a 74 section in RWP_settings.cfg (it can be empty) and that players set the slider.